.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
```

Comment:
CAPI also has an associated GitHub action that updates generated code, I believe we need something equivalent here https://github.com/kubernetes-sigs/cluster-api/blob/main/.github/workflows/dependabot.yml